### ISSUE TITLE
'visual sequence inspection evidence' review

### ISSUE THREAD

**Issue by beckyjackson** [evidenceontology/evidenceontology#242]
'visual sequence inspection evidence' (ECO:0001828) is a child of 'inference from background scientific knowledge used in manual assertion'.

My proposal is to created  move this term to be a direct child of "inference from background scientific knowledge" and rename it to "inference from sequence inspection".

Additionally, all the "curator inference" terms have "used in automatic assertion" children. I'm not sure that this makes sense for curator inference terms. I think that they should all actually be manual assertion terms, since it's the curator making the inference.

To achieve this, we would obsolete all the "automatic" terms and merge the manual terms into the parent terms. 'curator inference' could get the axiom "used_in only 'manual assertion'" to say that it cannot be used in an automatic assertion.

By contrast, all "evidence based on computational logical inference" (brought up in #240) could be automatic assertion terms.

I would be happy to create a prototype branch to implement this so that we can review, but I'd like to hear others' thoughts first.

**Comment by jbmunro** [comment]
The definition for this term is wonky.  It discusses "used in manual assertion" yet it is not a typical "used in manual assertion" term.

I think 'inference from sequence inspection' is not good given the very narrow definition which is to inspection of only the promoter region.

'inference from sequence inspection' is a good term but could potentially cover a lot of different things: putential putative binding sites, alignment quality/issues, motif conservation, etc.

I agree with your point regarding automatically generated "used in automatic assertion" children - they are senseless in the context of 'curator inference'

James
________________________________

**Comment by beckyjackson** [comment]
Would it make sense to make a new "inference from sequence inspection" class under curator inference, and then move this term to be a child of that?

We could rename it "putative binding site identification by sequence inspection", or something along those lines.

I don't like having a single child under a parent class, but it leaves room for more sequence inspection terms in the future.

**Comment by beckyjackson** [comment]
Based on discussion in #111, it looks like we should keep the "raw" evidence codes and make "used in manual assertion" children for these terms. IMO, we should just not make automatic evidence children for the "curator inference" terms.

I might be wrong though, is there a case where a curator inference can be automatically asserted? I would think that would be considered a computational inference.

**Comment by ivanerill** [comment]
This term was not intended to be curator inference, but author inference. The idea was that sometimes authors will identify a promoter (or some other type of sequence feature) based on the background knowledge. It was intended to be the "human" equivalent of ECO:0000028 and its children, but it could conceivably be thought of closer to ECO:0000034.

**Comment by beckyjackson** [comment]
Thanks for the clarification @ivanerill 

In that case, does it make sense to do:
```
- author inference
    - author inference from sequence inspection (NEW)
        - putative binding site identification by sequence inspection (ECO:0001828)
```

**Comment by ivanerill** [comment]
Hi @beckyjackson ,

Yes, although I don't know if there is a need to make it specific to "binding site". It could be "sequence feature" identification, or just leave it with the new proposed parent term.

Ivan

**Comment by mgiglio99** [comment]
I haven't thought much about this yet, so just commenting after reading this thread - I'm worried about having this term specific for authors since I think there are people making primary annotations based on their own inspection of the sequences - they are not "authors" per se, I guess they are "curators". We did this all the time in our manual genome curation days. Do we need separate terms for curators and authors? Should we just have one term for a human doing this type of thing?

**Comment by ivanerill** [comment]
Hi @mgiglio99 ,

I don't know. In one case, an author is making a statement based on their prior knowledge and likely testing or otherwise leveraging that inference within a paper. In the other case, a curator is making an interpretation on an author's paper that the author themselves have not made. I agree both are human inferences, but I don't know if differential weights would be applied to them if somebody were to mine GOAs using these terms.

GO evidence codes have historically differentiated between "Author statement" and "Curator statement", but it might be time to reassess this.

Ivan

**Comment by mgiglio99** [comment]
The case I'm thinking of is more along the lines of the person doing the analysis and annotations all themselves. So if I sequence a genome and then do an analysis where I find a binding site upstream of a set of genes and I make that annotation - I think I might use 'motif discovery evidence' ECO:0005531 or even just 'multiple sequence alignment'  ECO:0005555. 
So, now we want a term to capture when an author has done this. I wonder about getting too granular under the author inference node, we might end up duplicating things and creating confusion. 
Same is true of things under curator inference. If the curator is making this call themselves, then it is their annotation and I'm not sure it should be under a curator node.
Not sure if I'm explaining myself clearly here or not...

**Comment by ivanerill** [comment]
Correct. This is something we discussed with @mchibucos a while ago. When you are doing the inference, you may very well know that you are using a specific tool ('multiple sequence alignment' ECO:0005555), but here we were talking about an author eyeballing it (presumably based on their own knowledge of the system). I don't know how frequent this is these days, but it has certainly been common practice among people working on transcriptional regulation to just state that there is a XXX-binding site upstream of a gene, referring you to a figure containing the sequence with the relevant elements boldfaced or underlined.

**Comment by beckyjackson** [comment]
Another question to bring up is the difference between 'author inference' and 'author statement'. I think we made 'author inference' not too long ago to distinguish between curators and authors, but it seems more accurate to say 'author statement'.

**Comment by beckyjackson** [comment]
Suggestion: merge `author inference` into `author statement`.
Move `author statement from published clinical study` under `author statement supported by traceable reference` (unrelated).

For the 'visual sequence inspection evidence' class:

```
- inferential evidence (ECO:0000361)
    - inference from visual sequence inspection (ECO:0001828)
    - curator inference (ECO:0000205)
    - computational inferential evidence (ECO:0000362)
```

The inference from visual sequence inspection class could be used for *both* author inferences and curator inferences. This will be a general class, not specific to promoter regions, so we will broaden the definition. If somebody needs the more specific term in the future, we can add it.

This should always be a manual assertion. We are discussing the location of 'computational inferential evidence' in #240.

@ivanerill - if this looks OK to you, we will go ahead with this.

**Comment by ivanerill** [comment]
I personally would not merge author and curator inferences. I believe they are qualitatively different for a reasoner. One is a statement made by the author, the other an inference made by the curator based on their interpretation of the author's results/statements.

Other than that, having `inference from visual sequence inspection (ECO:0001828)` within `inferential evidence (ECO:0000361)` sounds good.

>This will be a general class, not specific to promoter regions, so we will broaden the definition.

Depending on how it is defined, it risks being too generic to be useful. I would at least try to capture its original intended sense (inference of sequence _features_ from visual inspection) in the parent or a derived term.

**Comment by beckyjackson** [comment]
> I personally would not merge author and curator inferences.

We didn't want to merge the author and curator inference classes. Our plan was to merge 'author inference' into 'author statement', but keep 'curator inference' distinct under 'inferential evidence'. Then, the visual sequence class would be directly under 'inferential evidence' so that it could be used for both curators and authors. Do you think it would be more correct to have two separate visual sequence inspection classes? One for authors, and one for curators?

> Depending on how it is defined, it risks being too generic to be useful. I would at least try to capture its original intended sense (inference of sequence features from visual inspection) in the parent or a derived term.

Label: inference of sequence features from visual inspection
Broad synonyms: inference from visual sequence inspection, visual sequence inspection evidence
Definition: A type of inferential evidence in which sequence features are inferred based on visual inspection of the sequence.

**Comment by ivanerill** [comment]
>We didn't want to merge the author and curator inference classes. Our plan was to merge 'author inference' into 'author statement', but keep 'curator inference' distinct under 'inferential evidence'. Then, the visual sequence class would be directly under 'inferential evidence' so that it could be used for both curators and authors. Do you think it would be more correct to have two separate visual sequence inspection classes? One for authors, and one for curators?

OK. Misunderstood. Certainly, no problem with this being used from both ends.

>Label: inference of sequence features from visual inspection
Broad synonyms: inference from visual sequence inspection, visual sequence inspection evidence
Definition: A type of inferential evidence in which sequence features are inferred based on visual inspection of the sequence.

Sounds good.

**Comment by beckyjackson** [comment]
There hasn't been any more discussion on this issue, so I'm going to go ahead and implement this.

**Comment by beckyjackson** [comment]
Resolved by abad913
